doc/how_to_introduce_breaking_changes.md

```diff
@@ -0,0 +1,291 @@
+# How to introduce Breaking Changes in service command
+
+Azure CLI has bi-annual breaking change releases coinciding with Microsoft **Build** and **Ignite**. Limiting breaking changes to twice a year provides a stable experience for customers while being able to keep up to date with the latest versions of Azure CLI and plan accordingly for announced breaking changes.
+
+## Breaking Changes in Azure CLI
+
+A breaking change refers to a modification that disrupts backward compatibility with previous versions. The breaking changes could cause a customer's script or automation written in a previous version to fail.
+
+The common examples of breaking changes include:
+* Modifying the names of parameters/commands.
+* Modifying the input logic of parameters.
+* Modifying the format or properties of result output.
+* Modifying the current behavior model.
+* Adding additional verification that changes CLI behavior.
+
+To mitigate the impact of breaking changes, Azure CLI delays breaking changes and coordinates half-yearly **Breaking Change Releases** that bundle multiple breaking changes together. This approach helps users plan ahead and adapt to the modifications effectively.
+
+### Breaking Change Window
+
+The breaking change window is a designated sprint that **permits** the merging of service command breaking changes. Any Pull Request merged during this sprint will be included in the subsequent Breaking Change Release.
+
+The timing of the breaking change window in Azure CLI aligns with [Microsoft Build](https://build.microsoft.com/) and [Microsoft Ignite](https://ignite.microsoft.com/). You could find the next Breaking Change Release plan in our [milestones](https://github.com/Azure/azure-cli/milestones).
+
+> It's highlighted that the introduction of breaking changes is typically prohibited within non-breaking-change window, based on what we stated above for consistency and stable tooling experience.
+> 
+> Exceptions to this policy may be considered under extraordinary circumstances. We understand and would like to help out. There would be high-graded justifications required to provide the info Azure CLI can access.
+> 
+> Please note that providing the required info for assessment does not mean it will be assured to be green-lighted for breaking changes. Team will still make the decision based on the overall impact.
+
+### Pre-announce Breaking Changes
+
+All breaking changes **must** be pre-announced two sprints ahead Release. There are two approaches to inform both interactive users and automatic users about the breaking changes.
```

Comment:
All breaking changes **must** be pre-announced two sprints ahead of Breaking Change window release. It give users the buffer time ahead to mitigate for better command experience.

Reply:
Updated